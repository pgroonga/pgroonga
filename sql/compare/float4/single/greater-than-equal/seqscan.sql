CREATE TABLE ids (
  id real
);

INSERT INTO ids VALUES (1.1);
INSERT INTO ids VALUES (2.1);
INSERT INTO ids VALUES (3.1);

CREATE INDEX grnindex ON ids USING pgroonga (id pgroonga_float4_ops);

SET enable_seqscan = on;
SET enable_indexscan = off;
SET enable_bitmapscan = off;

EXPLAIN (COSTS OFF)
SELECT id
  FROM ids
 WHERE id >= (2.1::real);

SELECT id
  FROM ids
 WHERE id >= (2.1::real);

DROP TABLE ids;
