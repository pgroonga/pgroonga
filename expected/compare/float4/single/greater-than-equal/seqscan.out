CREATE TABLE ids (
  id real
);
INSERT INTO ids VALUES (1.1);
INSERT INTO ids VALUES (2.1);
INSERT INTO ids VALUES (3.1);
CREATE INDEX grnindex ON ids USING pgroonga (id pgroonga_float4_ops);
SET enable_seqscan = on;
SET enable_indexscan = off;
SET enable_bitmapscan = off;
EXPLAIN (COSTS OFF)
SELECT id
  FROM ids
 WHERE id >= (2.1::real);
          QUERY PLAN           
-------------------------------
 Seq Scan on ids
   Filter: (id >= '2.1'::real)
(2 rows)

SELECT id
  FROM ids
 WHERE id >= (2.1::real);
 id  
-----
 2.1
 3.1
(2 rows)

DROP TABLE ids;
